src/cpp/shared_core/json/Json.cpp

```diff
@@ -586,12 +586,6 @@ Object Value::getValue<Object>() const
    return copy;
 }
 
-template<>
```

Comment:
This method returns a pointer to stack-allocated memory (moral equivalent of `return std::string().c_str()`); given that it's unsafe I've decided to remove it. This didn't affect anything in open source; if something is effected in pro then I think we need to change those usages to be safer.

Reply:
I agree - I don't see a reason why the caller couldn't get the string value and call `c_str()` themselves.